[2023-01-24 02:42:42,948] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Leer_Datos_Respuesta_Nasa scheduled__2023-01-01T01:12:00+00:00 [queued]>
[2023-01-24 02:42:42,977] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Leer_Datos_Respuesta_Nasa scheduled__2023-01-01T01:12:00+00:00 [queued]>
[2023-01-24 02:42:42,978] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-01-24 02:42:42,978] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2023-01-24 02:42:42,978] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-01-24 02:42:43,044] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): Leer_Datos_Respuesta_Nasa> on 2023-01-01 01:12:00+00:00
[2023-01-24 02:42:43,091] {standard_task_runner.py:52} INFO - Started process 1405 to run task
[2023-01-24 02:42:43,096] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Respuesta_Nasa', 'Leer_Datos_Respuesta_Nasa', 'scheduled__2023-01-01T01:12:00+00:00', '--job-id', '219', '--raw', '--subdir', 'DAGS_FOLDER/00-ETL_spaceX.py', '--cfg-path', '/tmp/tmpaokch7yy', '--error-file', '/tmp/tmpx0hac855']
[2023-01-24 02:42:43,099] {standard_task_runner.py:80} INFO - Job 219: Subtask Leer_Datos_Respuesta_Nasa
[2023-01-24 02:42:43,308] {task_command.py:371} INFO - Running <TaskInstance: Respuesta_Nasa.Leer_Datos_Respuesta_Nasa scheduled__2023-01-01T01:12:00+00:00 [running]> on host fd9891b969b0
[2023-01-24 02:42:43,633] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=Respuesta_Nasa
AIRFLOW_CTX_TASK_ID=Leer_Datos_Respuesta_Nasa
AIRFLOW_CTX_EXECUTION_DATE=2023-01-01T01:12:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-01T01:12:00+00:00
[2023-01-24 02:42:43,635] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-01-24 02:42:43,636] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /tmp && head /tmp/response_20230101.txt']
[2023-01-24 02:42:43,706] {subprocess.py:85} INFO - Output:
[2023-01-24 02:42:43,713] {subprocess.py:92} INFO - ***tmpx_pbptge
[2023-01-24 02:42:43,715] {subprocess.py:92} INFO - pymp-igsi0xnb
[2023-01-24 02:42:43,715] {subprocess.py:92} INFO - response_20230101.txt
[2023-01-24 02:42:43,715] {subprocess.py:92} INFO - spacex_20230101.json
[2023-01-24 02:42:43,715] {subprocess.py:92} INFO - tmpeacsl2bx
[2023-01-24 02:42:43,716] {subprocess.py:92} INFO - tmphys8mrio
[2023-01-24 02:42:43,716] {subprocess.py:92} INFO - tmpnvc2r2ky
[2023-01-24 02:42:43,716] {subprocess.py:92} INFO - tmpx0hac855
[2023-01-24 02:42:43,725] {subprocess.py:92} INFO - Confirmaci√≥n de la NASA, pueden proceder
[2023-01-24 02:42:43,732] {subprocess.py:96} INFO - Command exited with return code 0
[2023-01-24 02:42:43,892] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Respuesta_Nasa, task_id=Leer_Datos_Respuesta_Nasa, execution_date=20230101T011200, start_date=20230124T024242, end_date=20230124T024243
[2023-01-24 02:42:44,009] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-24 02:42:44,190] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
